test/passing/js_source.ml.ref

```diff
@@ -3414,17 +3410,14 @@ module SSMap = struct
 end
 
 let ssmap =
-  (module SSMap
-  : MapT
+  (module SSMap : MapT
```

Comment:
Here, since `MapT` is modified by `with`, we need a break before `:`.

Reply:
This is intended, I'll add an option to control the whole PR